lms/djangoapps/bulk_email/tasks.py

```diff
@@ -400,14 +402,17 @@ def _get_source_address(course_id, course_title):
     """
     course_title_no_quotes = re.sub(r'"', '', course_title)
 
-    # For the email address, get the course.  Then make sure that it can be used
-    # in an email address, by substituting a '_' anywhere a non-(ascii, period, or dash)
-    # character appears.
-    from_addr = u'"{0}" Course Staff <{1}-{2}>'.format(
-        course_title_no_quotes,
-        re.sub(r"[^\w.-]", '_', course_id.course),
-        settings.BULK_EMAIL_DEFAULT_FROM_EMAIL
+    # Translators: Bulk email from address e.g. ("Physics 101" Course Staff)
+    course_staff_title = _('"{course_name}" Course Staff')
+
+    from_addr_format = course_staff_title + u' <{course_id}-{from_email}>'
```

Comment:
@cahrens 

I actually think it totally OK to concatenate here.

The result value should always be like the following:
 - 'Course Staff of CS101 <cs101-courses@edx.org>'
 - 'طاقم المساق <cs101-courses@edraak.org>'

In an Arabic email client the second line would display just fine i.e. the name part will be first, and then the email with the brackets <> follows.

One more thing, in the actual string, I think it is only valid to put the name then the email with the brackets <>, otherwise it is invalid AFAIK.